json-format.md

```diff
@@ -58,8 +58,8 @@ exceptions noted below.
 | String        | [string][JSON-String]
 | Integer       | [number][JSON-Number], only the `int` component is permitted
 | Binary        | [string][JSON-String], [Base64-encoded][base64] binary
-| URI-reference | [string][JSON-String]
-| Timestamp     | [string][JSON-String]
+| URI-reference | [string][JSON-String] following [RFC 3986][RFC3986]
+| Timestamp     | [string][JSON-String] following [RFC 3339][RFC3339] (ISO 8601)
```

Comment:
From what I see there seem to be small differences between RFC3339 and ISO 8601 ad per the _Usage_ section in https://en.wikipedia.org/wiki/ISO_8601. 

Is supporting them both desired? Should we be more specifics? Do SDKs need to detect and parse?

Reply:
RFC3339: " This document defines a date and time format for use in Internet
   protocols that is a profile of the ISO 8601 standard for
   representation of dates and times using the Gregorian calendar."

RFC3339 is ISO8601, with limits

Reply:
Oooh. Got it. Thank you!